package.json

```diff
@@ -43,7 +43,7 @@
     "@types/styled-components": "^5.1.26",
     "eslint": "8.27.0",
     "eslint-config-next": "13.0.3",
-    "prisma": "^4.10.1",
+    "prisma": "^4.11.0",
```

Comment:
Probably no problem, but any reason why this is upgraded?

Reply:
No there's no reason. It just gave med the latest version when i installed it